### PR TITLE
Exclude containing directory in S3 path

### PR DESCRIPTION
**Pull request by nicl**
This seems to fix things, but I suspect it could be made more robust.

### CODE REVIEW

File: src/upload.ts [hunk 1]
```diff
@@ -26,16 +25,16 @@ export const upload = async (
     manifest.projectName,
     manifest.buildNumber,
     action && action.action,
-    name
+    name,
   ]
-    .filter(_ => _ != null)
+    .filter((_) => _ != null)
```
Comment (nicl) [inline src/upload.ts]: Dammit, I thought I'd disabled all my formatting stuff.

**Review by AWare — APPROVED** [review]
👏

**Comment by AWare** [comment]
@alexduf you experienced this bug as well, would this fix it for you?

**Review by alexduf — APPROVED** [review]

**Comment by alexduf** [comment]
Probably! I'll check next time we upgrade our build pipeline

**Pull request merged**